### PR TITLE
add favicons

### PR DESCRIPTION
**Pull request by marcobiedermann**
- apple touch icon
- windows web icon
- windows desktop icon
- chrome icon

### CODE REVIEW

File: _layouts/index.html [hunk 1]
```diff
@@ -33,6 +46,29 @@
...
+        function loadLanguage(lang) {
+            if (lang === {{ page.lang | jsonify }}) {
+                return;
+            } else if (lang === "en") {
+                window.location.replace("index.html");
+            } else {
+                window.location.replace("index_" + lang + ".html");
+            }
+        }
+    </script>
```
Comment (mikemcquaid) [inline _layouts/index.html]: What's this it for?

**Comment by marcobiedermann** [comment]
I've merged the master into my feature/favicon branch. Before we could not auto merge because the index page got updated in the meantime

**Comment by mikemcquaid** [comment]
Ah, ok. Could you try rebasing on `master` and see if that clears up the diff? Thanks!

**Comment by marcobiedermann** [comment]
Does this work for you? I pulled the latest master into my forked one and merged these changes into my favicons pull-request. Is that ok?

**Comment by mikemcquaid** [comment]
Still looks a bit weird. I'd recommend adding the main repository as a Git remote named e.g. `homebrew` and then doing `git fetch homebrew; git rebase homebrew/master` on your local branch and then `git push --force` to push your changes up.

**Comment by marcobiedermann** [comment]
ok, now I did what you suggested

**Comment by marcobiedermann** [comment]
I think my git knowledge is at the end. Sorry for the trouble, I am not a that big git pro. The basic command just worked fine for my daily coding :)

**Comment by mikemcquaid** [comment]
No worries! Can you try and manually just remove the added `<script>`? Thanks!

**Pull request merged**